ELIXIR.md

```diff
@@ -15,24 +15,24 @@
 
 ## Coding Guidelines
 
-Make sure you comply with [elixir style guide](https://github.com/rrrene/elixir-style-guide). In order to automate the style checking, install [Credo](https://github.com/rrrene/credo) and run it before creating a new Pull Request. If possible, always run credo with `strict` flag: `mix credo --strict`.
+Make sure you comply with this [elixir style guide](https://github.com/rrrene/elixir-style-guide). In order to automate the style checking, install [Credo](https://github.com/rrrene/credo) and run it before creating a new Pull Request. If possible, always run credo with the `strict` flag: `mix credo --strict`.
 Another must-have is to enable [Credo CI](http://credoci.inakalabs.com/) to automatically check your updates in GitHub.
 
 ## Build Tools
 
-Before pushing a new Pull Request, run your unit tests locally and never push them if something is not green. If the project is open-source, setup [Travis CI](https://travis-ci.org) to automatically run your unit tests after each new change. Remember to add a badge to your README file informing the build is passing.
+Before pushing a new Pull Request, run your unit tests locally and never push them if something is not green. If the project is open-source, setup [Travis CI](https://travis-ci.org) to automatically run your unit tests after each new change. Remember to add a badge to your README file informing visitors that the build is passing.
```

Comment:
I found 3 misspelled words in this sentence: 
- README
	- *Suggested Replacements:* redame
- CI
	- *Suggested Replacements:* cm, cr, fi, cfi, cie, ti, ca, ki, cf, bi, cit, ck, yi, cp, pi, cpi, c, ic, cu, gi, ni, co, ui, ct, cd, li, cir, cli, cy, si, cig, csi, sci, pci, cs, cc, ai, cia, cai, ch, xi, hi, civ, cif, chi, cb, vi, cg, mi, cl, di, cq, cv, cid, i, ii, ce, wi, cis
- open-source